### PR TITLE
add doc on OpenAPI build workflow

### PR DESCRIPTION
**Pull request by tomkralidis**
Adds/updates build instructions (consistent to https://github.com/opengeospatial/ogcapi-coverages/blob/master/standard/openapi/README.md)

cc @jerstlouis

### CODE REVIEW

File: openapi/README.md [hunk 1]
```diff
@@ -1,11 +1,23 @@
-# OpenAPI building blocks
+# OpenAPI definitions
 
-This example API definition can be used to provide an OpenAPI 3.0 definition for an implementation of _OGC API - Maps_.
+This example API definition can be used to provide an OpenAPI 3.0 definition for an implementation of _OGC API - Maps.
```
Comment (jerstlouis) [inline openapi/README.md:3]: This breaks the italic, could you please remove this line change?

**Review by jerstlouis — CHANGES_REQUESTED** [review]

File: openapi/README.md (hunk 1 above)
Comment (tomkralidis) [reply openapi/README.md]: Fixed.

**Comment by jerstlouis** [comment]
Thanks @tomkralidis .
I don't have merge rights currently on this repo, so cc'ing @joanma747 .

**Comment by tomkralidis** [comment]
Thanks @jerstlouis; @joanma747 let me know if there are any updates required.

**Comment by jerstlouis** [comment]
@tomkralidis There doesn't seem to be any GitHub action included in this PR, is something missing?

**Comment by tomkralidis** [comment]
@jerstlouis the initial PR was for dev workflow.  Good point, GitHub Actions workflows now added.

**Comment by jerstlouis** [comment]
@tomkralidis Thanks Tom! cc @joanma747

**Pull request merged**